src/sonic-device-data/tests/permitted_list

```diff
@@ -24,6 +25,7 @@ ext_tcam_freq
 force_core_pll
 fpem_mem_entries
 higig2_hdr_mode
+ifp_inports_support_enable
```

Comment:
Why this change is needed?  Same thing for below checks.

Reply:
I will remove it

Reply:
updated in https://github.com/Azure/sonic-buildimage/pull/5424/commits/2d79cb570370808e69015139681fe907667c81a8